client/src/components/FilmDetail.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import { useParams } from 'react-router-dom';
+import {useState, useEffect} from 'react';
+
+
+
+
+//const ViewContact = ({contact, toUpdate, toDelete})
```

Comment:
minor note: it's usually preferred to remove commented out code when you're committing to master